src/Sylius/Component/Core/OrderProcessing/PaymentProcessor.php

```diff
@@ -43,15 +38,11 @@ public function __construct(FactoryInterface $paymentFactory)
     /**
      * {@inheritdoc}
      */
-    public function createPayment(OrderInterface $order)
+    public function processOrderPayments(OrderInterface $order)
```

Comment:
also the name implies multiple payments but this method only creates a single new payment for the total of an order. Could be something like `PaymentFactory::createForOrderTotal()` ?